source/blog/_posts/2018-11-30-how-to-use-phive.md

```diff
@@ -0,0 +1,74 @@
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<phive xmlns="https://phar.io/phive">
+  <phar name="php-cs-fixer" version="^2.12.2" installed="2.13.0" location="./tools/php-cs-fixer" copy="true"/>
+  <phar name="shopwarelabs/psh" version="^1.2.0" installed="1.2.0" location="psh" copy="true"/>
+  <phar name="phpstan" version="^0.10.1" installed="0.10.3" location="./tools/phpstan" copy="true"/>
+  <phar name="shopwarelabs/sw-cli-tools" version="^0.0.1" installed="0.0.1" location="./tools/sw" copy="true"/>
+</phive>
+```  
+
+This sounds very interesting isn't it? But how difficult is it to create a new software project and use phive for managing all dev tools?
+
+## Setup a new Project with phive
+The question is very easy to answer, it is as easy as to create a new composer project. As a preparation we have to install phive on our system. To do it use the snippet below:
+
+```bash
+wget -O phive.phar https://phar.io/releases/phive.phar
+wget -O phive.phar.asc https://phar.io/releases/phive.phar.asc
+gpg --keyserver pool.sks-keyservers.net --recv-keys 0x9D8A98B29B2D5D79
+gpg --verify phive.phar.asc phive.phar
+chmod +x phive.phar
+sudo mv phive.phar /usr/local/bin/phive
+```
+
+After the installation we can easily install php-cs-fixer with ```phive install php-cs-fixer```. In this example php-cs-fixer is an alias and corresponds to the GitHub repository [FriendsOfPHP/PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer). You can find alternative installation instructions below:
+
+```bash
+phive install php-cs-fixer
+phive install shopwareLabs/sw-cli-tools
+phive install https://phar.phpunit.de/phpunit-4.8.6.phar
+```
+
+You can use the already mentioned alias, or an github repository with a maintained release section or the direct url to the specific file.  
+
+## Own phar files
+To install own phar files with phive you have several ways to achieve this: 
+
+* GitHub Releases (phive install username/project)
+* registered alias on phar.io (phive install alias)
+* explicit url (phive install https://server/file-1.0.0.phar)
+
+We will have a more detailed look in the GitHub Releases way and we will use [psh](https://github.com/shopwareLabs/psh) an example for that. We have the following requirements:
+
+* GPG Key for your project
+* GPG Key must be public available on the SKS Keyservers 
+* already generated phar file.
+
+If all requirements fit we open the console and go to our project directory where the phar file is stored. Now we will use this command ```gpg -u psh@example.com --detach-sign --output psh.phar.asc psh.phar``` to sign our generated phar file. After that we have a new file psh.phar.asc. Both files has to be uploaded to our existing version under https://github.com/shopwareLabs/psh/releases. That was all the work we needed for that. 
```

Comment:
Please add a sentence to describe where to store the `.asc` file if the phar file is installed per full url, or will the file signature than not be checked?